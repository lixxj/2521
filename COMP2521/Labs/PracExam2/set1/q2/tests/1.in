from: 2 , to: 3
2 1
2 3





