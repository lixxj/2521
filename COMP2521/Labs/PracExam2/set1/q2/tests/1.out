hasPath, From: 2 , To: 3 

Edge inserted:  2 -> 1 
Edge inserted:  2 -> 3 


hasPath: 1 

